### ISSUE TITLE
BTP ARV Vest - Ready to use

### ISSUE THREAD

**Issue by BigCaseQB** [LondonsBlueLine/tracker#675]
I have retextured (already in game) existing Vest 66 (From HANAKO's Vest & Earpiece EUP Pack) to a BTP Version as there is currently no BTP AFO vest in game nor available to download.
I have set it up to replace Texture 1 as Texture 0 is in use. Texture 2 is a SHERRIF texture and so obviously not used.

Harley and I have created a new EUP for this vest as well which will enable BTP to actually have a decent Outfit selection.
In these images I show the Current Vest 66 Texture 0 as well as the new texture 1.
I will also show the Suggested EUP outfit for BTP AFO with Vest 66 Texture 0 and Texture 1. Texture 1 should be used when implemented but Texture 1 is just so you can get an idea of the way it would look.

Pictures:

VEST PICS
(https://github.com/LondonsBlueLine/tracker/assets/139172405/faa7ecb9-e2b1-4498-ac6f-eccf6d5204f4)

VEST PICS
(https://github.com/LondonsBlueLine/tracker/assets/139172405/f56c9c3a-a81f-4c02-b8fd-887a1351e1b0)

VEST PICS
(https://github.com/LondonsBlueLine/tracker/assets/139172405/fa65ae41-c691-4afe-b378-cf7f7166a053)

VEST PICS
(https://github.com/LondonsBlueLine/tracker/assets/139172405/ee5a1b4a-024b-4603-b4be-f32c6b7c7f11)

EUP PICS
(https://github.com/LondonsBlueLine/tracker/assets/139172405/786bd7aa-1131-437f-a74f-b81afd3cc56d)

EUP PICS
(https://github.com/LondonsBlueLine/tracker/assets/139172405/53ac3433-3975-4018-9c66-64bfeee8134f)

Texture File (In JPEG form as cant upload a YTD to Github):
![BTP VEST PROTOTYPE 2 ](https://github.com/LondonsBlueLine/tracker/assets/139172405/0a062e08-252a-4553-a6d6-08d09bb4499e)

Any issues lmk.
-Casey AF26

**Comment by CalLBLRP** [comment]
Hiya @BigCaseQB,

Just to confirm this is a vest we already have in-game correct?

Cheers,
Cal.

**Comment by BigCaseQB** [comment]
> Hiya @BigCaseQB,
> 
> Just to confirm this is a vest we already have in-game correct?
> 
> Cheers, Cal.

Yes this is for existing vest 66. The vest 66 Texture 1 is currently in use so the idea is to replace Texture 2 (Sherriff) with this one and use it for BTP EUPs

**Comment by CalLBLRP** [comment]
No worries, could you also attach the link on where you found this free asset please?

**Comment by BigCaseQB** [comment]
I made it myself
I cannot upload the file to GitHub due to file type. Is there any other way I can send it to you? I provided the flat image texture above, if you want to do it that way?

**Comment by CalLBLRP** [comment]
No worries mate, if you made this yourself. That is perfectly fine. However, I will just need to ask you to provide some proof/evidence that you asked the creator of the original asset/vest permission to edit their work. 

Cheers,
Cal.

**Comment by BigCaseQB** [comment]
@CalLBLRP 
> No worries mate, if you made this yourself. That is perfectly fine. However, I will just need to ask you to provide some proof/evidence that you asked the creator of the original asset/vest permission to edit their work.
> 
> Cheers, Cal.

This vest is the HANAKO EUPVEST&EARPIECE 
https://www.lcpdfr.com/downloads/gta5mods/character/25692-eup-vest＆earpiece/ 

As per the description the retexturing and **release** of such textures is permitted as long as credit is given to HANAKO, it is not used for profit, only the texture is edited (not the model) and it is only released via LSPDFR and or GTA MODS.com.
As this texture is not being release the latter do not apply and we will be adhering to the "only-texture-files-edited" rule for retextures.
Credit, Non-Profit and The release on specific sites **do not apply** as we are not releasing the vest for public download.
If we wanted though we could go the extra mile and credit him somehow anyway - however as we haven't credited him for our current use of his work this shouldn't be a problem?

In short HANAKO vests are free and unrestricted to retexture... as long as the model isn't being altered (which we aren't) and that if we release it for public download (which we aren't). The Credit, Non-profit and Release locations rules are adhered to.

**Comment by CalLBLRP** [comment]
Perfect, thank you for that mate. As you can understand, this type of information is extremely crucial for us. This was not intended to come off rude or questionable in anyway. We just need to ensure we have permission to edit the assets that come into the server. 

Cheers,
Cal.

**Comment by BigCaseQB** [comment]
@CalLBLRP 
Yes of course, Apologies if mine came off confrontational… I was just trying to make it as clear as possible and show my thinking.
Did you want me to send over the YTD via some means?

**Comment by harleyBTP** [comment]
Evening @BigCaseQB, This has been put to dev mate and i believe some uniforms have been made, im not sure when this will be pushed but we are getting towards getting it made. 
thanks 
harley 
Sargant

**Comment by harleyBTP** [comment]
hiya mate update: 
these have been put towards the Quality check team to see if they are suitable i will get back to you with a further update matey

**Comment by BigCaseQB** [comment]
Any progress? @harleyBTP @CalLBLRP

**Comment by CalLBLRP** [comment]
It has passed Quality Check, just finishing off some touches with the development sided stuff and will look at getting it into main server in the next few weeks.
